core/src/main/java/org/springframework/security/authentication/UsernamePasswordAuthenticationToken.java

```diff
@@ -32,6 +32,8 @@
  * <code>String</code>.
  *
  * @author Ben Alex
+ * @author Norbert Nowak
+ * @since 5.7
```

Comment:
The `@since 5.7` belongs on the methods that you added. Please remove it from the class level and add it to `authenticated` and `unauthenticated`.